code/allmall.pro.user

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
this file as i know should be ignored because of gitignore. something happend and it was not ignored. exclude from commit pls